task.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir -p task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
При использовании флага `-p` можно не создавать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```